Assets/Scripts/Audio/AudioWrapper.cs

```diff
@@ -10,12 +9,13 @@ namespace Audio
     public class AudioWrapper : EverlastingSingleton<AudioWrapper>
     {
         [SerializeField] private List<SoundData> allSoundData;
-        private Dictionary<string, SoundData> _soundDict;
+        private readonly Dictionary<string, SoundData> _soundDict = new();
 
         private bool _dictionaryInitialised;
 
-        private void Start()
+        protected override void Awake()
```

Comment:
If I use Start, then the sound dictionary may not be prepared by the time another component is Starting and the file may not exist